next.clayui.com/content/docs/components/toolbars/management-toolbar.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Management Bar'
+description: 'Management toolbar is an extension of Toolbar. It is a combination of different components, including filters, orders, search, visualization select, and other actions that let users manage a dataset.'
+packageNpm: '@clayui/management-toolbar'
+sibling: 'docs/components/toolbars/css-management-toolbar.html'
+---
+
+import {ManagementToolbar, MinimalManagementToolbar, ResultsBar} from '../../../../src/components/clay/ManagementToolbar';
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+-   [API](#api)
+    -   [ManagementToolbar](#managementtoolbar)
+	-   [ItemList](#itemlist)
+	-   [Item](#item)
+	-   [ResultsBar](#resultsbar)
+		-   [ResultsBar.Item](#resultsbar.item)
+	-	[Search](#search)
+
+</div>
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon/">Lexicon</a>{' '}
+	<a href="https://liferay.design/lexicon/core-components/toolbars/management-bar/">
+		Management Toolbar Pattern
+	</a>{' '}
+	for a more in-depth look at the motivations and proper usage of this
+	component.
+</div>
+
+With ClayManagementToolbar you can create a bunch of Management Toolbar possibilities, use `ClayManagementToolbar` component for wrapping the management toolbar, `ClayManagementToolbar.ItemList` for creating groups and `ClayManagementToolbar.Item` for create items inside groups. Use `expand` property on `ClayManagementToolbar.ItemList` for expanding the item list.
+
+Use `ClayManagementToolbar.Search` for creating a Search input inside the management toolbar.
+
+<MinimalManagementToolbar />
+
+Use `ClayManagementToolbar.ResultsBar` for creating a results bar and `ClayManagementToolbar.ResultsBarItem` for defining items inside Results Bar.
```

Comment:
Maybe add some headers here? to separate the examples:

```suggestion
## ResultsBar

Use `ClayManagementToolbar.ResultsBar` for creating a results bar and `ClayManagementToolbar.ResultsBarItem` for defining items inside Results Bar.
```

Reply:
After some internal conversations, we decided to left as is